src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/resource/PrisonerAlertsController.kt

```diff
@@ -143,4 +149,62 @@ class PrisonerAlertsController(val alertService: AlertService) {
     )
     prisonNumbers: Collection<String>,
   ): Map<String, List<Alert>> = alertService.retrieveAlertsForPrisonNumbers(prisonNumbers)
+
+  @Operation(
+    summary = "Create an alert",
+  )
+  @ApiResponses(
+    value = [
+      ApiResponse(
+        responseCode = "201",
+        description = "Alert created successfully",
+        content = [Content(schema = Schema(implementation = Alert::class))],
+      ),
+      ApiResponse(
+        responseCode = "400",
+        description = "Bad request",
+        content = [Content(schema = Schema(implementation = ErrorResponse::class))],
+      ),
+      ApiResponse(
+        responseCode = "401",
+        description = "Unauthorised, requires a valid Oauth2 token",
+        content = [Content(schema = Schema(implementation = ErrorResponse::class))],
+      ),
+      ApiResponse(
+        responseCode = "403",
+        description = "Forbidden, requires an appropriate role",
+        content = [Content(schema = Schema(implementation = ErrorResponse::class))],
+      ),
+      ApiResponse(
+        responseCode = "409",
+        description = "Conflict, the person already has an active alert using the supplied alert code",
+        content = [Content(schema = Schema(implementation = ErrorResponse::class))],
+      ),
+    ],
+  )
+  @PostMapping("/{prisonNumber}/alerts")
+  @ResponseStatus(HttpStatus.CREATED)
+  @PreAuthorize("hasAnyRole('$ROLE_ALERTS_WRITER', '$ROLE_ALERTS_ADMIN', '$UPDATE_ALERT', '$ROLE_NOMIS_ALERTS')")
+  @UsernameHeader
+  @SourceHeader
+  fun createAlert(
```

Comment:
Tiny point here. The Swagger docs will use this in the URL and as there are two, it'll use createAlert and createAlert_1. I think there's another instance of it in the migrate alert controller. Maybe `createPrisonerAlert`?